types/gapi.auth2/index.d.ts

```diff
@@ -186,6 +186,11 @@ declare namespace gapi.auth2 {
          */
         hosted_domain?: string | undefined;
 
+        /**
+         * Enable or disable the use of browser FedCM APIs during sign-in.
```

Comment:
I think it would be nice to include the default value (`true`) in the JSDoc

Reply:
Thanks for the comment. I updated the JSDoc with more details on the use_fedcm behavior.